### PR TITLE
Userguide and editNote bugs

### PR DESCRIPTION
**Pull request by Nazryl**
- [x] #172
- [x] #168
- [x] #161
- [x] #149

### CODE REVIEW

File: src/main/java/seedu/notus/ui/Formatter.java [hunk 1]
```diff
@@ -112,14 +112,14 @@ public static String formatNotes(String header, ArrayList<Note> notes, Notebook
...
+        String header = message.concat(note.getTitle() + " " + note.getTagsName());
 
         if (note.getPinned()) {
-            header = header.concat(" | ISPIN: " + note.getPinnedString());
+            header = header.concat("| ISPIN: " + note.getPinnedString() + " ");
         }
 
         if (note.getIsArchived()) {
-            header = header.concat(" | ISARCHIVE: " + note.getIsArchived());
+            header = header.concat("| ISARCHIVE: " + note.getIsArchived());
```
Comment (R-Ramana) [inline src/main/java/seedu/notus/ui/Formatter.java:122]: Just to confirm, getPinnedString returns a string value and getIsArchived returns a boolean value. Just raising this to confirm your intention was to use getPinnedString and not getIsPinned? 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -199,6 +200,8 @@ Format: `edit-n /i INDEX [/t TITLE] ([/add INDEX STRING] OR [/ln LINE_INDEX CONT
 - Existing values will be updated to the input values.
 - When editing tags, the existing tags will be removed, while the non-existing tags will be added.
 
+💡 Content must have at least 1 line.
```
Comment (R-Ramana) [inline docs/UserGuide.md:203]:
```suggestion
💡 Note content must have at least 1 line.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -125,6 +125,7 @@ Enter Note:
 `Line 2`<br>
 `/end`
 
+💡 Content must have at least 1 line.<br>
```
Comment (R-Ramana) [inline docs/UserGuide.md:128]:
```suggestion
💡 Note content must have at least 1 line.<br>
```

**Review by R-Ramana — APPROVED** [review]

File: src/main/java/seedu/notus/ui/Formatter.java (hunk 1 above)
Comment (Nazryl) [reply src/main/java/seedu/notus/ui/Formatter.java:122]: Yes confirm. Open to changes. Both relay the same message to let the user know its pinned and archived.

Comment (R-Ramana) [reply src/main/java/seedu/notus/ui/Formatter.java:122]: In that case, might be better to standardize to have both returning the boolean value.

**Pull request merged**